dds_cli/data_putter.py

```diff
@@ -421,41 +419,41 @@ def add_file_db(self, file):
 
         return added_to_db, error
 
-    def update_project_size(self):
-        """Update the project size in one go via a API request."""
-
-        size_updated, error = (False, "")
-
-        # Perform request to DDS API
-        try:
-            response = requests.put(
-                DDSEndpoint.PROJECT_SIZE,
-                params={"project": self.project},
-                headers=self.token,
-                timeout=DDSEndpoint.TIMEOUT,
-            )
-        except requests.exceptions.RequestException as err:
-            # Log warning if error
-            # TODO (ina): Add the info to the error log if this happens --> can update manually
-            error = str(err)
-            LOG.warning(error)
-
-        # Verify ok response
-        if not response.ok:
-            error = f"Failed to update project: {response.text}"
-            LOG.exception(response.text)
-        else:
-            # Get response from endpoint
-            try:
-                json_resp = response.json()
-            except simplejson.JSONDecodeError as err:
-                LOG.warning(str(err))
-            else:
-                updated = json_resp.get("updated")
-                error = json_resp.get("error")
-
-                # TODO (ina): Add the info to error log if any error happens --> update manually
-                if not updated:
-                    LOG.warning(f"The project size could not be updated! Error: {error}")
-                else:
-                    LOG.info("Project size updated successfully!")
+    # def update_project_size(self):
```

Comment:
I'd say you should delete this and not comment it out. It's in git if one needs it.